lighthouse-cli/cli-flags.ts

```diff
@@ -56,6 +56,7 @@ export function getFlags() {
           ],
           'Configuration:')
       .describe({
+        'enable-error-reporting': 'Enables error reporting (on by default)',
```

Comment:
maybe we can come up with a different flag name, as it isn't really the case that it's on by default (instead it prompts by default), and a first glance may alarm some